src/test/java/uk/gov/hmcts/ccd/domain/types/DocumentValidatorTest.java

```diff
@@ -176,16 +173,14 @@ public void shouldValidateIfPortsAreSpecifiedAndMatch() {
     }
 
     @Test
-    public void shouldNotValidateIfPortsAreSpecifiedAndNotMatch() {
+    public void shouldValidateIfPortsAreSpecifiedAndNotMatch() {
         final DocumentValidator validatorWithPort = buildDocumentValidator("https://ng.reform.hmcts.net:7789");
```

Comment:
I think shouldValidateIfPortsAreSpecifiedAndNotMatch() and shouldValidateIfPortsAreSpecifiedAndMatch() are not relevant anymore. buildDocumentValidator does not use the url.

Reply:
Removed the tests. You're right.